src/Native/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -67,6 +92,12 @@ REDHAWK_PALEXPORT bool REDHAWK_PALAPI PalInit()
     if (!PalQueryProcessorTopology())
         return false;
 
+    g_flsIndex = FlsAlloc(FiberDetachCallback);
```

Comment:
Please add a comment about what we're doing with FLS here.  Basically something along the lines of "we use fiber detach callbacks to run our thread shutdown code because the fiber detach callback is made without the OS loader lock" is what I'd like to see.